src/neuroglancer/datasource/deepzoom/log.md

```diff
@@ -0,0 +1,73 @@
+`OffscreenCanvas` needs relatively recent Firefox (105, 2022-09-20), and brand new Safari (16.4, 2023-03-27 - which was 4 days ago at the time of writing).  
+`raw` encoding is removed now. While `createImageBitmap()` could handle whatever image format the browser supports, the filtering is kept in place and `png`, `jpeg`, and `jpg` formats are accepted only.
+
+4. Compatibility version
+
+This one works on older Firefox and Safari versions.  
+
+The built-in `png` and `jpeg` decoders are brought into action. As there are known Deep Zoom tile generators putting an extra `overlay` pixel row on right/bottom edge tiles, the specify-image-dimensions-beforehand requirement is solved with actually reading into the files (`PNG` has its dimensions at fixed positions, and in case of `JPEG` the first occurrence of a `SIFn` chunk is assumed to tell the full dimensions).  
+Extra quirk is that the `JPEG` decoder produces planar image (there is an explicit `transposeArray2d()` call for that in `decode_jpeg.ts`), while `PNG` returns packed (like `getImageData()` in the previous variant).
+
+5. Formatting, and an `export`
+
+Gulp is somewhat happy now.  
+Side note: it reformatted 178 files, 2 of them being mine (`base.ts` and `register_defaults.ts` were untouched).  
+
+`backend.ts`:
+
+* it wanted to add an extra empty line after `import`s, I'm not sure about that
+* it wanted to reformat the simple/future tile loader (`OffscreenCanvas` and co.) in the comment, that doesn't make sense
+* I personally prefer keeping `requestAsyncComputation()` parameters in their current form: the first line is for the RPC call (function, token, transferrables), and the second line is for the actual call parameters.
+
+`frontend.ts`:
+
+* I think it's fully gulp-accepted now (moved a single comment around after the change)
+* 3 utility functions are removed and `imported` from `precomupted` instead. One of them was not `export`ed, now it is. These may be good candidates to be collected at some completely different location anyway, with other utility functions.
+
+Random things:
+
+* Noticed the latest feature addition, the "hidden placeholder resolution" thing. It's not in use by this datasource yet, but definitely something to look into
+* While I needed the name distinctions to have a better track of what's local and what's imported, on a longer run it may be possible the inherit from `precomputed` (instead of what's happened here, so making a copy of it)
```

Comment:
Ultimately there should be very little in common, so better to have it independent as you've done.